documentation/host/error-monitoring.rst

```diff
@@ -6,13 +6,13 @@ Error monitoring
 When you run a FlexMeasures server, you want to stay on top of things going wrong. We added two ways of doing that:
 
 - You can connect to Sentry, so that all errors will be sent to your Sentry account. Add the token you got from Sentry in the config setting :ref:`sentry_access_token` and you're up and running! 
-- Another source of crucial errors are things that did not even happen! For instance, a task to import prices from a day-ahead market, which you depend on later for scheduling. We added a new CLI task called ``flexmeasures monitor tasks``, so you can be alerted when tasks have not successfully run at least so-and-so many minutes ago. The alerts will also come in via Sentry, but you can also send them to email addresses with the config setting :ref:`monitoring_mail_recipients`.
+- Another source of crucial errors are things that did not even happen! For instance, a task to import prices from a day-ahead market, which you depend on later for scheduling. We added a new CLI task called ``flexmeasures monitor latest-run``, so you can be alerted when tasks have not successfully run at least so-and-so many minutes ago. The alerts will also come in via Sentry, but you can also send them to email addresses with the config setting :ref:`monitoring_mail_recipients`.
```

Comment:
Can we add a paragraph explaining the `flexmeasures monitor last-seen` CLI command, too? It's not trivial how one would effectively use it. For example, if you want to monitor individual (bot) users, you might first want to be able to filter users by creating a custom user role, e.g. for having signed up to a monitoring service.

(Also, the latest-run CLI task isn't really `new` anymore.)